### PR TITLE
handle pdfs and heics

### PR DESCRIPTION
**Pull request by SmithPeder**
## Will now support HEIC images and PDF files

This has been kind of a pain in the ass for some time, as people love to upload these kinds of files.

Solves #5 

image

### CODE REVIEW

**Review by LudvigHz — APPROVED** [review]
LGTM

File: kaaf/handler.py [hunk 1]
```diff
@@ -54,25 +62,73 @@ def footer(self):
...
+    imgByteArr = imgByteArr.getvalue()
+    return imgByteArr
+
+
 def create_image_file(image):
-    if not "image/" in image:
+    """
+    Take an image in BASE64 format and return a NamedTemporaryFile containing the image.
+    Will handle PNG, JPEG and GIF without any changes, as FPDF will handle those files
+    without problem. For PDFs
```
Comment (erlingfn) [inline kaaf/handler.py:76]: Is there a missing sentence here?

**Review by erlingfn — CHANGES_REQUESTED** [review]
Looks good, just one small change

File: kaaf/handler.py (hunk 1 above)
Comment (SmithPeder) [reply kaaf/handler.py:76]: yapp, nice catch

**Pull request merged**